assets/shared/helpers/player/index.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Internal dependencies
+ */
+import * as videoFileAdapter from './video-file-adapter';
+import * as videopressAdapter from './videopress-adapter';
+import * as youtubeAdapter from './youtube-adapter';
+import * as vimeoAdapter from './vimeo-adapter';
+
+const VIDEO_TYPE = videoFileAdapter.ADAPTER_NAME;
+const VIDEOPRESS_TYPE = videopressAdapter.ADAPTER_NAME;
+const YOUTUBE_TYPE = youtubeAdapter.ADAPTER_NAME;
+const VIMEO_TYPE = vimeoAdapter.ADAPTER_NAME;
+
+const adapters = {
```

Comment:
It had a refactor, extracting all the player adapters from this file to external files.